pyrex.py

```diff
@@ -220,6 +228,9 @@ def build(args):
             except subprocess.CalledProcessError:
                 return 1
         else:
+            print(textwrap.fill("Running outside of Docker. No guarantees are made about your Linux " +
+                                "distribution's compatibility with Yocto."))
+            print()
```

Comment:
We have to be really careful about outputting to stdout in non-error cases, since the output is sometimes parsed. I'm not sure if this is going to cause a problem, but I'd feel a lot better writing it to stderr.

Reply:
I'm rescinding that. Output during "pyrex build" is fine, it's output during "pyrex run" that should be avoided.